vs-extension.shared/DeployProvider/DeployProvider.cs

```diff
@@ -369,6 +372,28 @@ public async Task DeployAsync(CancellationToken cancellationToken, TextWriter ou
                     // deployment successful
                     await outputPaneWriter.WriteLineAsync("Deployment successful!");
 
+                    // Now deploying to the internal storage if any
+                    if (contents.Any())
+                    {
+                        MessageCentre.InternalErrorWriteLine("Deploying content files to internal storage");
+                        foreach(var file in  contents)
+                        {
+                            MessageCentre.InternalErrorWriteLine($"Deploying {file.EvaluatedInclude}");
+                            if(file.EvaluatedInclude.Contains(Path.DirectorySeparatorChar))
+                            {
+                                MessageCentre.InternalErrorWriteLine("File should not be a path, internal storage does not support folders. It will still try to deploy.");
```

Comment:
```suggestion
                                MessageCentre.InternalErrorWriteLine("File is included as a path, but internal storage does not support folders. Note: It will still try to deploy.");
```

Reply:
Suggested change is trying to convey the error rather than how to handle it (as convention), but if possible, an extra "suggestion" of how to handle the error would be advantagous.

Reply:
> Suggested change is trying to convey the error rather than how to handle it (as convention), but if possible, an extra "suggestion" of how to handle the error would be advantagous.

Yes, I was even thinking to replace the '\' by a dash and write it. And also mention that in the docs. I'm open for any improvement on that side. Note that you can also have a look at the 2 other PR, the nf-interpreter one and the nf-debugger one where the core code is located.

Reply:
Actually, this depends on what are the target capabilities. For ORGPAL boards (and others that offer options other than internal storage) these can be stored in SDCard or USB MSD, for example.
I would suggest a warning message with a generic warning about that being a path and the target device may not support that. Or similar.

Reply:
ok, so we need an additional setting on the nfproj file to specify which drive to use. By default, in this draft implementation, it's using the internal storage.
Now: what is the interest of being able to deploy on a SDCard or external storage when it's much easier to do it another way?

Reply:
Considering that this is target related, I would argue that this should go into user prefs and not project.
Apart from that, as this is the 1st pass, its more than OK to default to internal storage.

Reply:
To your last question: that's, again, hardware dependent. The device may not have internal storage but may have an SDCard. That should be a perfectly valid storage for files being served to HTTP server or used as resources in any application...

Reply:
How about `File is included as a path, but operation does not allow this. Note: It will still try to deploy.`